src/salaryAndBenefits/SalaryAndBenefits.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from "./salaryAndBenefits.module.css";
+import Text from "src/components/text/Text";
+import {SalaryAndBenefits as SalaryAndBenefitsPayload} from "studio/lib/payloads/salaryAndBenefits";
```

Comment:
We might want to revisit naming here.

I see blog uses BlogPage, though that be better for the component name. 🤔  Seeing as we'll add similar setups like Blog + Benefits, such as Customer cases, we should look at the naming structure. Let's tackle that in a separate PR.

Reply:
Could you add a github task for this, and perhaps set up a quick chat about it, say tomorrow?

Reply:
Done https://github.com/varianter/variant.no/issues/493